reflex/ivars/sequence.py

```diff
@@ -707,6 +707,19 @@ def contains(self, other: Any) -> BooleanVar:
         """
         return array_contains_operation(self, other)
 
+    def pluck(self, field: StringVar | str) -> ArrayVar:
+        """Pluck a field from the array.
+
+        Args:
+            field: The field to pluck from the array.
+
+        Returns:
+            The array pluck operation.
+        """
+        if field is None:
```

Comment:
Either remove this or add `None` to the field type union

Reply:
OK, I just removed it. It should anyway have resulted in an empty array for logical reasons.